extensions/vscode/src/types/messages/hostToWebviewMessages.ts

```diff
@@ -71,15 +72,15 @@ export type RefreshContentRecordDataMsg = AnyHostToWebviewMessage<
   HostToWebviewMessageType.REFRESH_CONTENTRECORD_DATA,
   {
     contentRecords: (ContentRecord | PreContentRecord)[];
-    selectedContentRecordName?: string | null;
+    deploymentSelected: DeploymentSelector | null;
   }
 >;
 export type RefreshConfigDataMsg = AnyHostToWebviewMessage<
   HostToWebviewMessageType.REFRESH_CONFIG_DATA,
   {
     configurations: Configuration[];
     configurationsInError: ConfigurationError[];
-    selectedConfigurationName?: string | null;
+    deploymentSelected: DeploymentSelector | null;
```

Comment:
Is this no longer optional?

Reply:
no longer needed. It is always updated within the deployment selected, as we are writing out the deployment record on update of the selected configuration.